api/maven-api-core/src/main/java/org/apache/maven/api/Session.java

```diff
@@ -56,9 +56,21 @@
     @Nonnull
     SessionData getData();
 
+    /**
```

Comment:
Javadoc additions to existing API are useful but can be included in a separate PR that is easier to approve.